credentialdigger/client.py

```diff
@@ -982,11 +983,16 @@ def scan_user(self, username, category=None, models=None, debug=False,
             # Get repo clone url without .git at the end
             repo_url = repo.clone_url[:-4]
             logger.info(f'{i}/{repos_num}) Scanning {repo.url}')
-            missing_ids[repo_url] = self._scan(repo_url, scanner,
-                                               models=models,
-                                               debug=debug,
-                                               similarity=similarity,
-                                               git_token=git_token)
+            try:
+                missing_ids[repo_url] = self._scan(repo_url, scanner,
+                                                   models=models,
+                                                   debug=debug,
+                                                   similarity=similarity,
+                                                   git_token=git_token)
+            except GitCommandError:
+                logger.info(f"{i}/{repos_num} Ignore {repo_url} "
+                            "(it can not be cloned)")
```

Comment:
please note all the strings are defined by single quotes (`'`) and not double ones (`"`) so we have to be consistent

Reply:
the logger is in an exception so it shouldn't be at info level.
Better to use either warning or error